<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>581</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <italic>false</italic>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>22</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>账号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>22</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="button_password">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>22</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>密码柜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_web">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>22</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>网站柜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_app">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>22</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>软件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_func">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>22</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>特色功能</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_word">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>22</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
